lib/rules/no-cross-package-imports.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const CROSS_PACKAGE_IMPORT = /^ckeditor5/;
```

Comment:
We should care about scoped imports (`@ckeditor/ckeditor5-*`), and non-scoped (the `ckditor5` package).